Index: HackUTA/wolfram_alpha.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\nimport os\nimport re\nimport json\nimport requests\nfrom flask import Flask, request, redirect\nfrom twilio.twiml.messaging_response import MessagingResponse\n\napp = Flask(__name__)\n@app.route('/sms', methods=['GET','POST'])\ndef wolfram():\n    #Interact with simple api from wolframalpha\n    number = request.form['From']\n    msg = request.form['Body']\n    #EX: https://api.wolframalpha.com/v1/result?i=What+is+the+stock+price+of+Twitter%3F&appid=DEMO\n    #TODO env variable\n    appid = \"KLEJ7L-X4Y9P4KLV5\"\n    r = \"http://api.wolframalpha.com/v1/result?appid={}&i={}%3f\".format(appid,msg.replace(' ','+'))\n    response = requests.get(r).content.decode('utf-8')\n\n    print(response)\n    out = MessagingResponse()\n    out.message(response)\n    return str(out)\n\nif __name__ == '__main__':\n    app.run(debug=True)\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- HackUTA/wolfram_alpha.py	(revision f65954973f84e6bc4ad56215da4a120ed97ccb3a)
+++ HackUTA/wolfram_alpha.py	(revision )
@@ -9,20 +9,20 @@
 app = Flask(__name__)
 @app.route('/sms', methods=['GET','POST'])
 def wolfram():
-    #Interact with simple api from wolframalpha
-    number = request.form['From']
-    msg = request.form['Body']
-    #EX: https://api.wolframalpha.com/v1/result?i=What+is+the+stock+price+of+Twitter%3F&appid=DEMO
-    #TODO env variable
-    appid = "KLEJ7L-X4Y9P4KLV5"
-    r = "http://api.wolframalpha.com/v1/result?appid={}&i={}%3f".format(appid,msg.replace(' ','+'))
-    response = requests.get(r).content.decode('utf-8')
+	#Interact with simple api from wolframalpha
+	number = request.form['From']
+	msg = request.form['Body']
+	#EX: https://api.wolframalpha.com/v1/result?i=What+is+the+stock+price+of+Twitter%3F&appid=DEMO
+	#TODO env variable
+	appid = "KLEJ7L-X4Y9P4KLV5"
+	r = "http://api.wolframalpha.com/v1/result?appid={}&i={}%3f".format(appid,msg.replace(' ','+'))
+	response = requests.get(r).content.decode('utf-8')
 
-    print(response)
-    out = MessagingResponse()
-    out.message(response)
-    return str(out)
+	#print(response)
+	out = MessagingResponse()
+	out.message(response)
+	return str(out)
 
 if __name__ == '__main__':
-    app.run(debug=True)
+	app.run(debug=True)
 
Index: .idea/codeStyleSettings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/codeStyleSettings.xml	(revision )
+++ .idea/codeStyleSettings.xml	(revision )
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectCodeStyleSettingsManager">
+    <option name="PER_PROJECT_SETTINGS">
+      <value />
+    </option>
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/dictionaries/khanguy.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/dictionaries/khanguy.xml	(revision )
+++ .idea/dictionaries/khanguy.xml	(revision )
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="khanguy" />
+</component>
\ No newline at end of file
Index: HackUTA/Hack.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from marketing_notifications_python.models import app_db\n\n\ndef send_message(self, to, message, image_url):\n    self.twilio_client.messages.create(\n        to = to,\n        from_ = phone_number(),\n        body = message,\n        media_url = image_url\n    )\n    @views.route('/', methods=['GET', 'POST'])\n    @views.route('notifications', methods=['GET', 'POST'])\n    def notifications():\n        form = SendMessageForm()\n        if request.method == 'POST' and form.validate_on_submit():\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- HackUTA/Hack.py	(revision f65954973f84e6bc4ad56215da4a120ed97ccb3a)
+++ HackUTA/Hack.py	(revision )
@@ -2,15 +2,15 @@
 
 
 def send_message(self, to, message, image_url):
-    self.twilio_client.messages.create(
-        to = to,
-        from_ = phone_number(),
-        body = message,
-        media_url = image_url
-    )
-    @views.route('/', methods=['GET', 'POST'])
-    @views.route('notifications', methods=['GET', 'POST'])
-    def notifications():
-        form = SendMessageForm()
-        if request.method == 'POST' and form.validate_on_submit():
+	self.twilio_client.messages.create(
+		to = to,
+		from_ = phone_number(),
+		body = message,
+		media_url = image_url
+	)
+	@views.route('/', methods=['GET', 'POST'])
+	@views.route('notifications', methods=['GET', 'POST'])
+	def notifications():
+		form = SendMessageForm()
+		if request.method == 'POST' and form.validate_on_submit():
 
